### PR TITLE
Add additional translations to trainerpage

### PR DESCRIPTION
**Pull request by pki0**
Try it :)

### CODE REVIEW

**Comment by friscoMad** [comment]
Inactive are mostly ok active are not working.

pokemad _ entrenadores_2017-03-18_14-18-25

File: core/process/aru.php [hunk 1]
```diff
@@ -654,7 +654,11 @@
 			$resultPkms = $mysqli->query($req);
 
 			while ($resultPkms && $dataPkm = $resultPkms->fetch_object()) {
+				$dataPkm->today = $locales->TODAY;
```
Comment (friscoMad) [inline core/process/aru.php:660]: Probably this should go as a property of the main response, but as it is an array maybe in per trainer and not per pokemon.

File: core/js/trainer.content.js [hunk 2]
```diff
@@ -245,13 +245,13 @@ function printPokemon(pokemon,pokeimg_suffix,iv_numbers){
 	}
 	trainerPokemon.append(progressBar);
 	if (pokemon.last_scanned === '0') {
-		trainerPokemon.append($('<small>',{text: "Today"}));
+		trainerPokemon.append($('<small>',{text: pokemon.today}));
```
Comment (friscoMad) [inline core/js/trainer.content.js:248]: Add spacing

**Comment by pki0** [comment]
Sould work now.

**Pull request merged**